### PR TITLE
fix: rerunning an old job cause task and hook execution time to differ

### PR DESCRIPTION
**Pull request by kushsharma**

### CODE REVIEW

**Comment by sravankorumilli** [comment]
As discussed, instead of storing the `execution_time` at task instance level let's just store it at job_run as that is our intention.
That means `instance` table will just be used for logging purpose.

`execution_time` here will means the first time job has been scheduled by the scheduler for a given run, it doesn't change on re runs, when working on the documentation lets capture that too. cc : @arinda-arif

File: models/job.go [hunk 1]
```diff
@@ -106,7 +106,7 @@ type JobSpecNotifier struct {
 }
 
 type JobSpecTask struct {
-	Unit     *Plugin
+	Unit     *Plugin `json:"-" yaml:"-"`
```
Comment (sravankorumilli) [inline models/job.go:109]: what scenarios this change will impact?

Comment (kushsharma) [reply models/job.go:109]: Test file where we marshal and unmarshal specs.

**Comment by sravankorumilli** [comment]
changes look good to me

File: models/job.go (hunk 1 above)
Comment (sravankorumilli) [reply models/job.go:109]: Alright

**Pull request merged**